R/colors.R

```diff
@@ -259,9 +264,24 @@ encode_hex <- function(values) {
   colors
 }
 
+# Convert HTML color keywords (plus "transparent") to integer matrix with 3
+# columns (r, g, b) and length(str) rows. Errors on invalid strings.
+decode_color_keyword <- function(str) {
+  color <- css_color_keywords[str]
+  if (anyNA(color)) {
+    stop("Invalid color keyword(s)")
```

Comment:
I think you only want to do this if `mustWork = TRUE`?

Reply:
No, the `decode_*` functions can be extremely strict about their input because they're protected by conservative regexes--in this case, `regex_color_keyword`.